discord/enums.py

```diff
@@ -425,6 +426,10 @@ class StickerType(Enum):
     apng = 2
     lottie = 3
 
+class InviteTarget(Enum):
+    unknown = 0
+    stream  = 1
```

Comment:
```suggestion
    stream  = 1
    embedded_application = 2
```

This will need documentation too.